<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MENU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);
font: 75 12pt &quot;Sitka Heading&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Heading</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Bot_Doctor">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(194, 236, 255);
font: 75 12pt &quot;Sitka Heading&quot;;</string>
       </property>
       <property name="text">
        <string>Doctor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bot_Paciente">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(194, 236, 255);
font: 75 12pt &quot;Sitka Heading&quot;;</string>
       </property>
       <property name="text">
        <string>Paciente</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bot_Salir">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(194, 236, 255);
font: 75 12pt &quot;Sitka Heading&quot;;</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
